contents/docs/api/post-only-endpoints.mdx

```diff
@@ -245,6 +245,100 @@ curl -v -L --header "Content-Type: application/json" -d '{
 }' https://app.posthog.com/batch/
 ```
 
+### Responses
+
+#### Status code: 200
+
+##### Responses
+
+```js
+{ 
+    status: 1 
+}
+```
+
+**Meaning:** A `200: OK` response means we have successfully received the payload, it is in the correct format, and the project API key (token) is valid. It **does not** imply that events are valid and will be ingested. As mentioned under [Invalid events](#invalid-events), certain event validation errors may cause an event not to be ingested. 
```

Comment:
_API key is valid_ or _API key is present_?

Reply:
Valid actually. We do validate this and send an error response if it isn't valid